Marlin/src/config/default/Configuration_adv.h

```diff
@@ -257,6 +257,21 @@
 #define FANMUX1_PIN -1
 #define FANMUX2_PIN -1
 
+/**
+ * Part-Cooling Fan M106/M107 Auto Selection
+ *
+ * This feature allows you to set the P value for M106/M107 
+ * automatically to the currend used extruder index
+ * e.g. on RADDS board are 2 PWM Controlled fan outs 
+ *      using a Dual Extruder solution,
+ *      at M106/M107 the P-value will be automatically set 
+ *      to the current Extruder index and the Extruder related
+ *      blower will cool the model. 
+ */
+
+// #define AUTO_FILAMENT_FAN_SELECTION 
```

Comment:
I'm not sure we should add this option. It breaks the expectation of the default behavior of `M106`/`M107`, and we're generally concerned about having some machines out there which do one thing, while other machines do another thing given the same command. It would be better to add a flag parameter like `M106 A` that would use the current extruder index, rather than a feature that alters the default behavior of `M106`/`M107` commands.

Reply:
Okay that is feasible and it make absolute sense to introduce a flag property instead of setting up the firmware that can not be managed on runtime. I will change it. Argument name A seems to be okay for me.

Reply:
moved into an other PR #12389